test/functional/rpc_psbt.py

```diff
@@ -19,6 +19,8 @@ class PSBTTest(BitcoinTestFramework):
     def set_test_params(self):
         self.setup_clean_chain = False
         self.num_nodes = 3
+       # TODO: remove -txindex. Currently required for getrawtransaction call.
```

Comment:
`getrawtransaction` is never called in this test, it seems

Reply:
Also, I don't think those TODOs should be added at all. They get added and never fixed, then get stale.

Regardless, fixing the TODO would require to implement some sort of (in-memory) database for transactions in some tests? Might as well just use the txindex feature of Bitcoin Core, with the benefit that it gets more test exposure.

Reply:
I disagree. `-txindex` is not how we recommend people run bitcoind. It's best that the functional tests reflect the way that we expect bitcoind to be run. Adding unusual options just to make the test run means that we could introduce a regression and it not get caught by the test suite (eg `getrawtransaction` stops working unless `-txindex` is set).

@amitiuttarwar has already stated that she intends to fix up the tests in a follow-up PR: https://github.com/bitcoin/bitcoin/pull/15159#discussion_r249332216

Reply:
Ah thanks, I guess it is fine to leave as is.